src/constants/routes.jsx

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
+export const DASHBOARD_TEACHER = '/teacher/dashboard';
```

Comment:
I don't think a separate route file is needed here. You can just use the path in the react router segment in App.jsx